### PR TITLE
Birchall fix

### PR DESCRIPTION
**Pull request by SimonGreenhill**

### CODE REVIEW

**Comment by xrotwang** [comment]
We did have a discussion on whether or not we want to allow changing identifiers (the `Birchall-2016-126` in this case) - although I can't find it right now (@LinguList do you know?).
I think we wanted to keep a list of errata, basically to create redirects in the web app, right?

**Comment by xrotwang** [comment]
Ah, we have this: https://github.com/clld/concepticon-data/blob/master/concepticondata/retired.json
So possibly this list should also include other identifier categories, like `ConceptList`.

**Comment by LinguList** [comment]
yes, will need to run through that thingy of retired. I also wonder how it's possible to have 1 merger now that the only merger we had before (mother and "mother ") is now deleted?

**Comment by xrotwang** [comment]
Where do you see the merger thing? Did you re-create the READMEs?

File: concepticondata/conceptlists/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 | [Bennet-1976-100](Bennet-1976-100.tsv) | 100 | 100 | 0 |
 | [Bennet-1976-50a](Bennet-1976-50a.tsv) | 50 | 100 | 0 |
 | [Bennet-1976-50b](Bennet-1976-50b.tsv) | 50 | 100 | 0 |
-| [Birchall-2016-126](Birchall-2016-126.tsv) | 126 | 100 | 1 |
+| [Birchall-2016-125](Birchall-2016-125.tsv) | 125 | 100 | 1 |
```
Comment (LinguList) [inline concepticondata/conceptlists/README.md]: @xrotwang, the merger is in this line, but it should not be there, as the former merger was MOTHER.

**Comment by SimonGreenhill** [comment]
sorry -- I hand-edited README.md, and didn't update that column

**Comment by xrotwang** [comment]
Ok, time to write the tutorial :)

**Comment by SimonGreenhill** [comment]
Ok, added to retired.json and reran `concepticon stats` (the latter is not showing up as a diff, as the hand-changes I made earlier capture that).

File: concepticondata/retired.json [hunk 2]
```diff
@@ -639,6 +639,11 @@
       "id": "Galicio-2015-28-28",
       "comment": "corrected typo",
       "replacement": "Galucio-2015-28-28"
+    },
+    {
```
Comment (LinguList) [inline concepticondata/retired.json]: You'll have to add replacments for each of the entries, I'm afraid, as the 2016-125 is now the new ID :(

**Comment by SimonGreenhill** [comment]
Ok, I'm adding an entry for each renumbering as:

```JSON
{
    "id": "Birchall-2016-126-29",
    "comment": "removed duplicate item in wordlist",
    "replacement": "Birchall-2016-125-28"
},
```

...but what should I do for the actual deleted item? (Birchall-2016-126-19): 

```JSON
{
    "id": "Birchall-2016-126-19",
    "comment": "removed duplicate item in wordlist",
    "replacement": none  # OR? 
},

```

**Comment by LinguList** [comment]
since it's duplicate, why not link to the source of the duplicate, i.e., its identical entry?

**Comment by LinguList** [comment]
In fact, @xrotwang uses these to create a forward-link to the new entry if you call for the olden one. So most straightforward approach would be to link this to the new unified entry now, I guess.

**Comment by SimonGreenhill** [comment]
how's this?

**Comment by SimonGreenhill** [comment]
Phew!

**Comment by LinguList** [comment]
Just discussed with @MacyL, who said she was going "through hell" with the Huang-1992-1820 (!!!) mapping: we need T-shirts with a print on them, saying "no matter what you link now" (front) "you'll never walk alone: Concepticon" (back).

**Comment by MacyL** [comment]
From someone is "going through hell".... 
front
back

**Pull request merged**